index.js

```diff
@@ -62,6 +63,16 @@ function recolorScrolledMenu() {
   }
 }
 
+//Hide play button and overlay on video and start playing video
```

Comment:
Awesome, So when the user hits the play button the overlay dissappears. Settings the autoplay source is a nice way to go about playing this when the overlay dissapears. The youtube api looks complicated and this is a nice workaround. 

Is it possible to create another function that replaces the overlay when the video pauses? My shallow dive into the youtube API says that would require a bit more work, so for now, I think it works as is, that the overlay only appears on pageload and then it dissapears until refresh. Great.